people/affiliations.json

```
@@ -191,4 +191,8 @@
         "name": "Roberto Scolaro",
         "company": "Sysdig"
     }
+    "sboschman": {
```

```suggestion
    },
    "sboschman": {
```

Otherwise it would become an invalid JSON